### ISSUE TITLE
Crash using Mupen64plus 2.0 and glide64mk2 (sorry I can't delete previous issue, which was incomplete)

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#585]
Originally reported on Google Code with ID 586

```
Using a 2010 Macbook Pro with 10.9 (64-bit)
Mupen64plus 2.0
Glide64mk2 video plugin

When playing majora's mask, every time I enter the inn via the front door (not the
top door) or speak to the owner the game quits. It just shuts down and I get a crash
report. This doesn't happen using the rice graphics plugin, but that one won't let
me use the inventory screen properly (all kinds of artifacts). 

Any ideas? I can change the config file for the glide plugin and I'm ok with taking
any hits on how the game looks, but there is no graphics interface for this atm so
i'm not sure what 1's and 0's to change around.

Thanks to anyone that responds!
```

Reported by `vmachiel` on 2013-11-28 12:55:32

**Comment by richard42** [comment]
```
I see this in my terminal after the crash. 

./run_glide.sh: line 1:   764 Segmentation fault: 11  ./mupen64plus.app/Contents/MacOS/mupen64plus
--corelib ./mupen64plus.app/Contents/MacOS/libmupen64plus.dylib --plugindir ./mupen64plus.app/Contents/MacOS
--gfx mupen64plus-video-glide64mk2 "$@"
logout

```

Reported by `vmachiel` on 2013-11-28 13:52:45

**Comment by richard42** [comment]
```
Can you please compile the glide64mk2 plugin with the make parameters

 OPTFLAGS="-O2" DEBUG=1

and run it with gdb (please create a backtrace where it crashes). For more information
about the compilation of mupen64plus-video-glide64mk2 under MacOS X please consult
https://bitbucket.org/richard42/mupen64plus-core/src/7974979802d39e2fd926dc9b1a102bb525b1cf41/tools/osx_build_instructions.txt?at=default
or try to contact Richard42 (for example using IRC irc.freenode.net channel #mupen64plus)
```

Reported by `sven@narfation.org` on 2013-11-28 18:06:54

**Comment by richard42** [comment]
Reported by `sven@narfation.org` on 2013-12-18 12:42:26
- **Status changed**: `Duplicate`
- **Merged into**: #579
